src/main/java/internetshop/dao/jdbc/ItemDaoJdbcImpl.java

```diff
@@ -26,20 +26,20 @@ public ItemDaoJdbcImpl(Connection connection) {
     public Item create(Item item) {
         String query = String.format(Locale.ENGLISH,
                 "INSERT INTO %s.items(name, price) VALUES ('%s', %.2f);",
-                dbName, item.getName(), item.getPrice());
+                DB_NAME, item.getName(), item.getPrice());
 
         try (Statement stmt = connection.createStatement()) {
             stmt.executeUpdate(query);
             return item;
         } catch (SQLException e) {
-            logger.warn("Can't write to BD", e);
+            logger.warn("Can't write item to BD" + item.getName(), e);
```

Comment:
```suggestion
            logger.warn("Can't write item to DB" + item.getName(), e);
```
база даних - database, тому краще скорочувати як DB